ethers-core/src/utils/anvil.rs

```diff
@@ -284,7 +284,7 @@ impl Anvil {
             }
         }
 
-        child.stdout = Some(reader.into_inner());
+        child.stdout = None;
```

Comment:
we don't need to even set it again, but I'd change how we take it to

`let stdout = child.stdout.take().expect("Unable to get stdout for anvil child process");`

Reply:
Yeah, much better - fixed